app/util/project_paths.py

```diff
@@ -5,6 +5,13 @@ def __get_jira_yml():
     return Path(__file__).parents[1] / "jira.yml"
 
 
+def __get_jira_result_dir():
+    return Path(__file__).parents[1] / "results" / "jira"
```

Comment:
you can make 1 method name it for example `get_artifacts_dir` and `use return os.environ.get('TAURUS_ARTIFACTS_DIR')` Also reuse it in conftest.py script